src/manage.py

```diff
@@ -217,8 +217,7 @@ def pull_registry_info(self):
     # Check if there's a NPM scope or the default scope @@npm.
     assert self.scope.startswith('@')
 
-    headers = {'Accept': 'application/vnd.npm.install-v1+json; q=1.0, application/json; q=0.8, */*'}
-    response = urlfetch.fetch(util.npm_registry_url(self.scope, self.package), headers=headers, validate_certificate=True)
+    response = util.registry_get(self.scope, self.package)
```

Comment:
There's no etag... it's really quite a terrible, undocumented API. There's no quota, rate limits so I think its ok. I believe the `_rev` will let us know if it has changed so at least we can guard processing/database hits.